pkg/hubble/exporter/exporter_test.go

```diff
@@ -65,7 +64,7 @@ func TestExporter(t *testing.T) {
 		assert.NoError(t, err)
 
 	}
-	//nolint: testifylint
+	// nolint: testifylint
```

Comment:
Is this intended? From https://golangci-lint.run/usage/false-positives/#nolint-directive:

> Use `//nolint` instead of `// nolint` because machine-readable comments should have no space by Go convention.